lib/msal-common/src/error/AuthError.ts

```diff
@@ -20,6 +20,7 @@ export const AuthErrorMessage = {
  */
 export class AuthError extends Error {
 
+    static AUTH_ERROR_NAME: string = "AuthError";
```

Comment:
I would suggest just `Name` as the static field for all classes. `AuthError.AUTH_ERROR_NAME` is redundant.